src/components/transport_manager/src/transport_manager_impl.cc

```diff
@@ -540,14 +540,19 @@ void TransportManagerImpl::AddConnection(const ConnectionInternal& c) {
   connections_.push_back(c);
 }
 
-void TransportManagerImpl::RemoveConnection(uint32_t id) {
+void TransportManagerImpl::RemoveConnection(
+    uint32_t id, transport_adapter::TransportAdapter* transport_adapter) {
```

Comment:
@vlantonov `In C, if a function needs to modify a variable, the parameter must use a pointer, eg int foo(int *pval). In C++, the function can alternatively declare a reference parameter: int foo(int &val).`